sphinxcontrib/images.py

```diff
@@ -297,47 +298,49 @@ def configure_backend(app):
             ).load()
         except StopIteration:
             raise IndexError("Cannot find sphinxcontrib-images backend "
-                                "with name `{}`.".format(backend_name_or_callable))
+                             f"with name `{backend_name_or_callable!r}`.")
     elif callable(backend_name_or_callable):
         backend = backend_name_or_callable
     else:
         raise TypeError("sphinxcontrib-images backend is configured "
                         "improperly. It has to be a string (name of "
                         "installed backend) or callable which returns "
-                        "backend instance but is `{}` (type:`{}`). Please read "
+                        "backend instance but is "
+                        f"`{backend_name_or_callable!r}`. Please read "
                         "sphinxcontrib-images documentation for "
-                        "more informations."
-                        .format(backend_name_or_callable,
-                                type(backend_name_or_callable)))
+                        "more information.")
 
     try:
         backend = backend(app)
     except TypeError as error:
-        logger.info('Cannot instantiate sphinxcontrib-images backend `{}`. '
-                 'Please, select correct backend. Available backends: {}.'
-                 .format(config['backend'], ', '.join(ep.name for ep in get_entry_points())))
+        logger.info('Cannot instantiate sphinxcontrib-images backend '
+                    f'`{config["backend"]}`. Please, select correct backend. '
+                    f'Available backends: {", ".join(ep.name for ep in get_entry_points())}.')
         raise SystemExit(1)
 
     # remember the chosen backend for processing. Env and config cannot be used
     # because sphinx try to make a pickle from it.
     app.sphinxcontrib_images_backend = backend
 
     logger.info('Initiated sphinxcontrib-images backend: ', nonl=True)
-    logger.info('`{}`'.format(str(backend.__class__.__module__ +
-                           ':' + backend.__class__.__name__)))
+    logger.info(f'`{backend.__class__.__module__}:{backend.__class__.__name__}`')
 
     def backend_methods(node, output_type):
         def backend_method(f):
             @functools.wraps(f)
             def inner_wrapper(writer, node):
                 return f(writer, node)
             return inner_wrapper
```

Comment:
This looks like this decorator does nothing and can probably be removed. I'm not sure if it did something in the past.